AAI/AAIConnectProfile.md

```diff
@@ -449,7 +452,9 @@ the Broker.
 
 5.  Claim Clearinghouses MUST provide protection against attacks as outlined in
     [RFC 6819](https://tools.ietf.org/html/rfc6819).
-    
+
+    1. Section 5.1.6 of RFC 6819 contains a SHOULD section that states `Ensure that client applications do not share tokens with 3rd parties.` This profile allows for Clearinghouses to consume access tokens from multiple brokers which are not considered 3rd parties. 
```

Comment:
Not sure about this feedback, so feel free to ignore this comment:

"which are not considered 3rd parties" could be more clear by stating something like

"This profile provides a mechanism for Clearinghouses to consume access tokens from multiple brokers in a manner that does not involve 3rd parties."

You can add a sentence about taking care to not spread the tokens to any other services that would be considered 3rd parties.